TWLight/templates/header_partial_b4.html

```diff
@@ -104,6 +104,12 @@
                 {% trans "Log out" %}
               </a>
             </li>
+          {% elif not user.is_authenticated %}
+            <li role="presentation" class="mobile-menu align-middle nav-item">
+              <a class="btn twl-btn" href="{% url 'auth_login' %}?next={{request.path}}">
```

Comment:
Instead of using the Django `auth_login` default view, can you please change it to `oauth_login`?

Reply:
is oauth login pre-created or do i have to create it?

Reply:
It already exists, you can see it being used here, for example: https://github.com/WikipediaLibrary/TWLight/blob/master/TWLight/templates/login_partial.html#L64